pkg/util/slice/slice.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2020 caicloud authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package slice
+
+// ContainsString is a helper func to judge whether a alice contains a element
+func ContainsString(slice []string, s string) bool {
```

Comment:
Define a string slice struct, and it has `Contains()` and `Remove()` methods.

Reply:
I don't think so, this is a `tools` function, maybe using `[]string` directly is simpler for users.